cgmml/data_utils/dataset_generation_hrnet3d/hrnet_dataset.py

```diff
@@ -44,13 +61,31 @@
         # Get data paths
         depthmap_file = (path + row[METADATA_DEPTHMAP]).replace('"', '')
         rgb_file = (path + row[METADATA_RGB]).replace('"', '')
+        json_file = (path + row[METADATA_RGB]).replace('"', '') + '-hrnet.json'
         output_file = f'{output_path}/{row[METADATA_ARTIFACT_ID]}.obj'.replace('"', '')
+        if os.path.isfile(output_file):
+            continue
 
-        # Process data
         try:
+            # Process data
             body = BodyPose.create_from_rgbd(depthmap_file, rgb_file, calibration_file)
             if (body.get_person_count() != 1):
                 continue
             body.export_object(output_file)
+
+            # Create anonymized raw dataset
+            image = body.get_child_image(True, False) * 255
+            image = image[..., ::-1].copy()
+            image = np.rot90(image, 1)
+            json_raw = (raw_path + row[METADATA_RGB]).replace('"', '') + '-hrnet.json'
+            depthmap_raw = (raw_path + row[METADATA_DEPTHMAP]).replace('"', '')
+            rgb_raw = (raw_path + row[METADATA_RGB]).replace('"', '')
+            mkdirs(depthmap_raw)
+            mkdirs(rgb_raw)
+            cv2.imwrite(rgb_raw + '.jpg', image)
+            os.rename(rgb_raw + '.jpg', rgb_raw)
```

Comment:
Prefer f-strings over `+`:
```suggestion
            os.rename(f'{rgb_raw}.jpg', rgb_raw)
```

Note that there are more occurrences above.